### DISCUSSION TITLE
Icon request: shamrock

### DISCUSSION THREAD

**Discussion by thepresident** [FortAwesome/Font-Awesome#21157] [Icon Requests]
Think of St Patrick's Day or Ireland. Definitely worth including and I'll be the first and surely not the only one to use!
![shamrock](https://cloud.githubusercontent.com/assets/5954452/3060989/855e2ed2-e1fe-11e3-8cc5-56cdf95250e0.jpg)

**Comment by stevenvh17** [comment]
Just wondering: isn't the four-leafed symbol more common?

**Comment by thepresident** [comment]
Hi Steven,

That's a very good point! The icon attached above is an example only, but I think it should be 3 leaves. If I'm not mistaken shamrock is a 3 leaf clover and it can only be with 3 leaves - anything with more or less leaves is not shamrock. Perfect example:http://www.aerlingus.com/ also some more details here: http://en.wikipedia.org/wiki/Shamrock

**Comment by PMMi11er** [comment]
+1

**Comment by tommyauskoeln** [comment]
Even after Paddys day 2015 I would love to see the shamrock :+1:

**Comment by gEndelf** [comment]
+1

**Comment by bobchauvin** [comment]
+1

**Comment by kwaaui** [comment]
+1

**Comment by alemilani** [comment]
+1

**Comment by Kellamity** [comment]
Both a 3 leaf to represent Irish things and a 4 leaf to represent luck would be good

**Comment by Owlscape** [comment]
I would love to get a 3 leaf clover and a 4 leaf clover option, +1
![Four+Leaf+Clover+Magnet_2](https://user-images.githubusercontent.com/42075430/114320533-d8c16380-9adb-11eb-921a-fe8668da8c37.jpg)

**Comment by sensibleworld** [comment]
We have this icon: https://fontawesome.com/icons/clover?s=solid

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
☘ [U+2618](https://emojipedia.org/emoji/%E2%98%98/) Shamrock
Also known as `Clover` and `Trefoil`.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because it already exists in Font Awesome Pro! [Take a look at it here.](https://fontawesome.com/icons/clover?f=classic&s=solid)

**Comment by thepresident** [comment]
That's a 4-leaf clover, not a shamrock. Shamrock is 3-leaf and generally represents Ireland. https://en.m.wikipedia.org/wiki/Shamrock

**Comment by sensibleworld** [comment]
Oops! You're right. Re-opening.
